eng/pipelines/templates/steps/analyze.yml

```diff
@@ -108,12 +108,6 @@ steps:
       artifactName: 'packages'
       targetPath: $(Build.ArtifactStagingDirectory)
 
-  - template: ../steps/run_apistub.yml
-    parameters:
-      ServiceDirectory: ${{ parameters.ServiceDirectory }}
-      TestMarkArgument: ${{ parameters.TestMarkArgument }}
```

Comment:
Do we need any of these other args in the new usages?

Reply:
We have removed this step in analyze and only service directory is required in build job. I will remove other params from template itself.

Reply:
Done.